libavcodec/hevcdec.c

```diff
@@ -2782,7 +2782,7 @@ static int hls_decode_entry_tiles(AVCodecContext *avctxt, void *input_tile, int
         int y_ctb = (ctb_addr_rs / s->ps.sps->ctb_width) << s->ps.sps->log2_ctb_size;
```

Comment:
Undo a change from FFMPEG (patch to allow 16+ HEVC threads) that didn't play well with Scotts Tiled HEVC changes. We can live with 16 threads for now (we ship with like, 2).